tweego_file_splitter.py

```diff
@@ -1,64 +1,112 @@
+'''
+Heavily based on tweego-file-splitter by Zachac :
```

Comment:
If I merge this in, it would become the new version, so no need for 'heavily based on' part.

Maybe can be a new description like, "Version 1.0.1" with the comments

Or perhaps this comment can be removed? I think it would be clear in the readme change log

Reply:
Just as I said, i'm not a real dev, I just get a github account primarly to help on some documentation. Again, feel free to ignore all my readme and arrange this how you want it.
I didn't think I could finish a proper version of the script, or to make a pull request of it.